kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -1,6 +1,124 @@
+window.SuperUserCreateModalView = BaseView.extend({
+    events: {
+        "click .create-btn": "create_superuser_click",
+        "keypress #id_superusername": "key_user",
+        "keypress #id_superpassword": "key_pass",
+        "keypress #id_superemail": "key_email"
+    },
+
+    template: HB.template("user/superusercreatemodal"),
+
+    initialize: function() {
+        _.bindAll(this);
+        this.render();
+        $("body").append(this.el);
+    },
+
+    render: function() {
+        this.$el.html(this.template());
+        _.defer(this.addSuperUserForm);
+    },
+
+    addSuperUserForm: function() {
```

Comment:
Let's use a consistent naming scheme (looks like the rest of it is snake cased) so `add_super_user_form`